public/app/percona/settings/components/Platform/Connected/Connected.tsx

```diff
@@ -49,21 +48,19 @@ export const Connected: FC = () => {
     closeModal();
     try {
       await PlatformService.forceDisconnect();
+      appEvents.emit(AppEvents.alertSuccess, [Messages.forceDisconnectSucceeded]);
+      setDisconnecting(false);
+      dispatch(fetchServerInfoAction());
+      dispatch(fetchSettingsAction());
     } catch (e) {
       // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
       const error = e as AxiosError<ForceDisconnectErrorBody>;
```

Comment:
What will happen here if error is not sended by backend but is created by failed connection for example? 
I can imagine something like:
     let message = null
     if(e instanceof of AxiosError<ForceDisconnectErrorBody>) {
        message = error.response?.data?.message;
     }
    
      logger.error(e);
      appEvents.emit(AppEvents.alertError, [message ?? 'Unknown error']);

      And we can even get rid of type assertion

Reply:
good point 👍